DOCS-2543 iOS Monitoring > Crash Reporting and Error Tracking Beta (#604)
DataDog/dd-sdk-ios: docs/rum_collection/_index.md

The RUM SDK automatically tracks user sessions depending on options provided at the SDK initialization. To add GDPR compliance for your EU users and other [initialization parameters][9] to the SDK configuration, see the [Set tracking consent documentation][8].

> <div class="alert alert-info"><p>Crash Reporting and Error Tracking is available in beta. [Setup here][10] </p>

Reviewer: ```suggestion
<div class="alert alert-info"><p>Crash Reporting and Error Tracking is available in beta. To sign up, see <a href="https://docs.datadoghq.com/real_user_monitoring/ios/crash_reporting">Crash Reporting (beta)</a>.</p>
```

)
``` 

> ## Crash reporting

Reviewer 2: ```suggestion
## iOS Crash Reporting and Error Tracking
```

[7]: https://docs.datadoghq.com/account_management/api-app-keys/#client-tokens
[8]: /real_user_monitoring/ios/advanced_configuration/#set-tracking-consent-gdpr-compliance
[9]: /real_user_monitoring/ios/advanced_configuration/#initialization-parameters

> [10]: /real_user_monitoring/ios/crash_reporting

Reviewer: ```suggestion
```
